app/support/auto_logout.rb

```diff
@@ -13,11 +15,16 @@ def perform
 
   def order_details
     OrderDetail.purchased_active_reservations.
-      where("reservations.actual_end_at IS NULL AND reserve_end_at < ?", Time.zone.now - 1.hour).
+      where("reservations.actual_end_at IS NULL AND reserve_end_at < ?", Time.zone.now).
```

Comment:
Also, for I switched the method name to `Reservation.relay_in_progress` so not to get that confused with reservation-only reservations.